src/log.rs

```diff
@@ -204,7 +205,8 @@ where
 {
     let template = FilterLogEventsInputBuilder::default()
         .log_group_name(&args.group)
-        .log_stream_names(&args.stream)
+        // clap ensures that this option is present unless --tail is passed
+        .log_stream_names(args.stream.as_deref().unwrap())
```

Comment:
from https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_FilterLogEvents.html, I think, it's possible to use some regex that matches with everything like `"."`. Not sure do you need it or not. I don't have use AWS at  the moment and cannot check and change it myself